packages/@divvi/mobile/src/walletConnect/request.ts

```diff
@@ -60,10 +60,10 @@ export function* handleRequest(
 
   switch (method) {
     case SupportedActions.eth_signTransaction: {
-      if (!serializableTransactionRequest) {
-        throw new Error('preparedTransaction is required when using viem')
+      if (!serializableTransactionRequests || serializableTransactionRequests.length === 0) {
+        throw new Error('preparedTransactions is required when using viem')
       }
-      const tx = getPreparedTransaction(serializableTransactionRequest)
+      const tx = getPreparedTransaction(serializableTransactionRequests[0])
```

Comment:
Do we ever expect the # of transactions to be anything other than exactly 1 for these cases? sendCalls should be the only case that has >1 TX, right?

Reply:
I guess it just seems strange that the types allow for there to be multiple TXs in these cases, and we would just drop any after the first, though maybe not worth worrying about.